### PR TITLE
update log when image changes during migration, make it info when not…

### PR DESCRIPTION
**Pull request by haibinxie**
… running on airgaped env.

Updates to dryrun log:
1. split log into multiple lines
2. if image is different, only warn if running on air gapped env.
3. filter some more expected env var and volume differences found during test

### CODE REVIEW

File: pkg/dryrun/dryrun.go [hunk 1]
```diff
@@ -41,6 +41,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/yaml"
 
+	"github.com/google/uuid"
```
Comment (pure-jliao) [inline pkg/dryrun/dryrun.go:44]: nit: external import in second group

**Review by pure-jliao — APPROVED** [review]
LGTM

File: pkg/dryrun/validation.go [hunk 2]
```diff
@@ -227,6 +243,7 @@ func (d *DryRun) deepEqualEnvVars(env1, env2 []v1.EnvVar) error {
 			"NODE_NAME":            true,
 			"PX_NAMESPACE":         true,
 			"PX_SECRETS_NAMESPACE": true,
+			"controller_sn":        true,
```
Comment (dsupure) [inline pkg/dryrun/validation.go:246]: What is controller_sn? Why it's all lowercase?

Comment (haibinxie) [reply pkg/dryrun/validation.go:246]: it's an env variable required by ccm container, daemonset spec gen just pass through

**Review by piyush-nimbalkar — APPROVED** [review]

**Pull request merged**